pkg/webhook/server.go

```diff
@@ -246,6 +250,15 @@ func (s *DefaultServer) Start(ctx context.Context) error {
 	idleConnsClosed := make(chan struct{})
 	go func() {
 		<-ctx.Done()
+
+		// Disable HTTP keep-alives to close persistent connections after next
+		// HTTP request. Clients may reconnect until routes are updated and
+		// server listeners are closed however this should start gradually
+		// migrating clients to server instances that are not about to shutdown
+		srv.SetKeepAlivesEnabled(false)
+		// Wait before shutting down webhook server
+		<-time.After(s.Options.ShutdownDelay)
```

Comment:
```suggestion
		time.Sleep(s.Options.ShutdownDelay)
```
We don't need to create a channel, I think.

Reply:
Haha oops, fixed